src/granite/associations.cr

```diff
@@ -52,26 +52,20 @@ module Granite::Associations
   end
 
   macro has_many(model)
-    {% if model.is_a? TypeDeclaration %}
-      def {{model.var}}
-          Granite::AssociationCollection(self, {{model.type}}).new(self)
-      end
-    {% else %}
-      def {{model.id}}
-          Granite::AssociationCollection(self, {{model.id.camelcase}}).new(self)
-      end
-    {% end %}
+    def {{model.id}}
+      Granite::AssociationCollection(self, {{model.id.camelcase}}).new(self)
+    end
   end
 
-  macro has_many(model, through)
-    {% if model.is_a? TypeDeclaration %}
-      def {{model.var}}
-          Granite::AssociationCollection(self, {{model.type}}).new(self, {{through}})
-      end
-    {% else %}
-      def {{model.id}}
-          Granite::AssociationCollection(self, {{model.id.camelcase}}).new(self, {{through}})
-      end
-    {% end %}
+  macro has_many(model, model_name)
```

Comment:
`model` is no longer really descriptive here, is it more of the name of the getter?